### PR TITLE
Add CI for all workflow charts

### PR DESCRIPTION
**Pull request by gabriel-farache**
Closes https://issues.redhat.com/browse/FLPATH-1689

Add CI for all workflow charts to check that the helm charts, once deployed, are working fine

### CODE REVIEW

File: .github/workflows/create-ocp-project.yaml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+          kubectl get sfp -A
+          kubectl wait --for=condition=Ready=true pods -l "app.kubernetes.io/name=backstage" --timeout=10m
+          kubectl get pods -o wide
+          kubectl wait --for=condition=Ready=true pods -l "app=sonataflow-platform" --timeout=180s
+
+      - name: Deploy MTA serverless workflow and its components
+        run: |
+          TARGET_NS=sonataflow-infra
+          cd charts
+          helm install create-ocp-project create-ocp-project -n "${TARGET_NS}"
```
Comment (masayag) [inline .github/workflows/create-ocp-project.yaml:43]: I'm not sure this workflow can be tested on k8s.
AFAIK, the Project crd shouldn't be available on k8s.
Is the purpose is to test the installation only? or also the invocation?
Or should only the workflows with script under e2e folder are tested for their execution?

Comment (gabriel-farache) [reply .github/workflows/create-ocp-project.yaml:43]: The purpose is to test only the installation
The workflows for which we want to test the execution, like M2K, are named `*-e2e` (for now only M2K, even MTA is not testing e2e, like in the serverless-workflow repo, I created an issue for that https://issues.redhat.com/browse/FLPATH-1830)

**Review by ElaiShalevRH — APPROVED** [review]

**Comment by gabriel-farache** [comment]
@ElaiShalevRH You do have merge permissions, right?

**Comment by ElaiShalevRH** [comment]
> @ElaiShalevRH You do have merge permissions, right?

No

**Comment by gabriel-farache** [comment]
> > @ElaiShalevRH You do have merge permissions, right?
> 
> No

You have a pending invitation to join the repo as writer, please check your email (and spam too maybe?)

**Pull request merged**